pkgs/by-name/ka/kazumi/package.nix

```diff
@@ -1,33 +1,31 @@
 {
   lib,
   fetchFromGitHub,
-  flutter324,
+  flutter327,
```

Comment:
optional: consider moving this to all-packages.nix as per https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name#changing-implicit-attribute-defaults